build.txt

```diff
@@ -1,6 +1,6 @@
 ﻿displayName = Longbows FTW
 author = gardenapple
-version = 1.12.2
+version = 2.0.0
```

Comment:
Why the major version change? We're not breaking major features here

Reply:
Just wanted it to be distinct. I'll change the second number if that is bothersome. I am aware that it is SemVer.